include/SFML/Audio/SoundStream.hpp

```diff
@@ -180,6 +180,11 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
 protected:
 
+    enum
+    {
+        NoLoop = -1 ///< "Invalid" endSeeks value, telling us to continue uninterrupted
+    };
```

Comment:
AFAIK the only advantage to using enum in this context is to avoid ODR violations.